frontend/src/pages/Report.tsx

```diff
@@ -232,6 +232,11 @@ export const Report = () => {
     }
     const unsavedEntries = [];
     for await (let entry of newTimeEntries) {
+      if (typeof entry.hours === "string") {
+        entry.hours === ""
+          ? (entry.hours = 0)
+          : (entry.hours = parseFloat(entry.hours));
+      }
```

Comment:
I think you should try to convert it to a number and then work out the logic from there.

Reply:
> I think you should try to convert it to a number and then work out the logic from there.

Do you mean to show the converted value in the cell rather than modifying the entered string using regular expressions and _then_ convert it?

That depends on what the converted value looks like for varying input. It could potentially result in surprises, such as being able to enter `1e10` or `-Inf` etc., depending on the capabilities of the parser.